app/tr_php/01/task02.php

```diff
@@ -12,8 +12,12 @@
  * ・コネクト太郎さんに、「おはようございます！！」と挨拶をするメッセージを、$message_oneに代入してください。
  * ・コネクト次郎さんに、「おやすみなさい！！」と挨拶をするメッセージを、$message_twoに代入してください。
 */
-$message_one;
-$message_two;
+function getGreetingMessage($name, $greeting) {
+    return "$greeting, $name!";
+}
```

Comment:
1．指定した規約と異なっている部分があります。
関数の開始括弧は、単独行である必要があります。

2．文字列の変数を埋め込むときは`{}`で囲むようにしましょう。どこまでが変数か分かりやすくなります。
```suggestion
function getGreetingMessage($name, $greeting)
{
    return "{$greeting}, {$name!}";
}
```

Reply:
ご指摘、ありがとうございます！
関数の開始括弧は、単独行である必要があることについて、気づきませんでした。
また、文字列の変数を埋め込むときは{}で囲むようにしますと、確かにわかりやすくなると思います。
今後、上記の２点を意識しながら関数を書くようにします。